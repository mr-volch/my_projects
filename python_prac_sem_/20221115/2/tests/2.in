class C:
  num = Num()

print(C().num)
a = C()
a.num = 5
d = C()
d.num = 11
print(d.num+a.num)
a.num = ('cur','cur')
print(a.num+d.num)
d.num = range(1, 1, 3)
print(d.num, a.num)
