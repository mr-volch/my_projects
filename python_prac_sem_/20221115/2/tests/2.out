0
16
13
0 2
